src/main/java/donk/Ui.java

```diff
@@ -0,0 +1,112 @@
+package donk;
+
+import donk.task.Task;
+import donk.task.TaskList;
+import donk.task.TaskType;
+
+/**
+ * Handles all user interactions
+ */
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    /**
+     * Displays an error message indicating that no save file was found
+     * and a new task list will be started.
+     */
+    public void showLoadingError() {
+        System.out.println("No save file found, starting new task list");
+    }
+
+    /**
+     * Greets the user with a welcome message when the application starts.
+     */
+    public void greet() {
+        String greetMsg = " ____________________________________________________________\n"
+                + " Hello! I'm donk.Donk, the super intelligent chatbot\n"
+                + " What can I do for you?\n"
+                + "____________________________________________________________\n";
```

Comment:
Consider creating a drawLine() function that returns the separating line as a String. Would help cut down on a lot of repeated code.